Makefile

```diff
@@ -165,9 +165,9 @@ build_python: .make/build_python
 		rm -rf ./bin/ ../python.bin/ && cp -R . ../python.bin && mv ../python.bin ./bin && \
 		rm ./bin/go.mod && \
 		python3 -m venv venv && \
-		./venv/bin/python -m pip install build==1.2.1 && \
+		./venv/bin/python -m pip install build==1.2.1 wheel && \
```

Comment:
Just leaving this in. If we take https://github.com/pulumi/ci-mgmt/pull/1691 then this will remain, and if we don't it will get removed next ci-mgmt update.